content/flutter-notification/email-and-slack-notifications.md

```diff
@@ -41,4 +43,4 @@ To receive a notification when a build starts, check the checkbox **Notify when
 
 When you set up email or Slack publishing, Codemagic publishes the following artifacts:
 
-- `app`, `ipa`, `apk`, the archive with Flutter web build directory, Linux application bundle files, Windows MSIX packages
+- `app`, `ipa`, `apk`, the archive with Flutter web build directory, Linux application bundle files
```

Comment:
There's a pending fix for publishing MSIX packages too.

Reply:
Thanks, @helinanever. I make changes to this PR. The issue has been fixed.